src/main/java/seedu/address/model/book/Rate.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.book;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a book's rate.
+ * Guarantees: immutable.
+ */
+public class Rate {
```

Comment:
Rename this class to `Rating`. (Shift+F6 or Refactor -> Rename)
Don't forget to change variables and descriptions to `rating`.